core/src/main/scala/org/http4s/headers/Expires.scala

```diff
@@ -18,8 +13,14 @@ object Expires extends HeaderKey.Internal[Expires] with HeaderKey.Singleton {
   *
   * @param expirationDate the date of expiration
   */
-final case class Expires(expirationDate: HttpDate) extends Header.Parsed {
-  val key = `Expires`
-  override val value = Renderer.renderString(expirationDate)
-  override def renderValue(writer: Writer): writer.type = writer.append(value)
-}
+final case class Expires(expirationDate: java.time.Instant)
+object Expires {
+  implicit val expiresHeader: Header[Expires] =
```

Comment:
I was thinking about going full cats-convention on the arbitraries.  Maybe we should start that here?

Reply:
As in put them in their own package so I don't need to write them in every spots companion?